flake.nix

```diff
@@ -194,6 +201,7 @@
                 username = "jdoe";
                 homeDirectory = "/home/jdoe";
               };
+              stylix.enable = true;
```

Comment:
This is required as [stylix/#244][1] does not inherit the home-manager enable status properly.

[1]: https://github.com/danth/stylix/pull/244

Reply:
It was moved to be toggled by the `jhome.styling` option.